portals/developer-portal/src/defaultContent/styles/mcp-landing.css

```
@@ -0,0 +1,538 @@
+  gap: 0.4375rem;
+  border-radius: 1.25rem;
+  padding: 0.3125rem 0.75rem;
+  font-size: 0.75rem;
+  font-weight: 600;
+}
+.mc-manage-status-badge::before {
+  content: '';
+  display: inline-block;
+  width: 0.4375rem; height: 0.4375rem;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)